packages/main/test/pages/Input.html

```diff
@@ -338,6 +338,18 @@ <h3>sap_horizon</h3>
 
 	<br>
 	<br>
+
+	<div id="xss-wrapper">
+		<ui5-button id="xss-btn">add xss item</ui5-button>
+		<ui5-input show-suggestions id="xss-input">
+			<ui5-suggestion-item text=""></ui5-suggestion-item>
+			<ui5-suggestion-item text="<b></b>"></ui5-suggestion-item>
+			<ui5-suggestion-item text="[b]test[/b]"></ui5-suggestion-item>
+			<ui5-suggestion-item text="[[[b]]]"></ui5-suggestion-item>
```

Comment:
I suggest you try adding a test case that I know your unit test will fail on so that you can see the issue in your highlight logic.
```suggestion
			<ui5-suggestion-item text=""></ui5-suggestion-item>
			<ui5-suggestion-item text="<b></b>"></ui5-suggestion-item>
			<ui5-suggestion-item text="[b]test[/b]"></ui5-suggestion-item>
			<ui5-suggestion-item text="[[[b]]]"></ui5-suggestion-item>
			<ui5-suggestion-item text="&amp;"></ui5-suggestion-item>
```